spec/models/metrics/moves/counts_by_time_bin_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Metrics::Moves::CountsByTimeBin do
+  subject(:metric) { described_class.new }
+
+  it 'includes the BaseMetric module' do
+    expect(described_class.ancestors).to include(Metrics::BaseMetric)
+  end
+
+  it 'initializes label' do
+    expect(metric.label).to eql(described_class::METRIC[:label])
+  end
+
+  describe 'calculate' do
+    let(:yesterday) { Metrics::TimeBins::COMMON_TIME_BINS.find { |x| x.title == 'yesterday' } }
+    let(:next_7_days) { Metrics::TimeBins::COMMON_TIME_BINS.find { |x| x.title == 'next 7 days inc today' } }
+
+    before do
+      create(:move, date: 4.days.ago)
+      create(:move, date: Date.yesterday)
+      create(:move, date: Date.today)
+      create(:move, date: Date.tomorrow)
+      create(:move, date: 4.days.from_now)
+      create(:move, date: 8.days.from_now)
+    end
+
+    it 'computes the metric' do
+      expect(metric.calculate('total', yesterday)).to be(1)
+      expect(metric.calculate('total', next_7_days)).to be(3)
```

Comment:
shouldn't this be 4 🤔

Reply:
nope - it should be 3, i.e.
* today + 
* tomorrow + 
* 4.days.from_now

it should _not_ include the move 8.days.from_now.

However, I think I will change the definition of next 7 days to exclude today, reducing this from 3 to 2.